data/simplified_procedural_codes.json

```diff
@@ -216,7 +216,12 @@
             "label": "Tomatoes",
             "commodities": [
                 "0702000007",
-                "0702000099"
+                "0702000099",
```

Comment:
These don't come back with a commodities page - they point to different commodity codes that are valid 1/25 onwards.